Corona-Warn-App/src/main/res/layout/fragment_onboarding.xml

```diff
@@ -2,6 +2,11 @@
 <layout xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:app="http://schemas.android.com/apk/res-auto"
     xmlns:tools="http://schemas.android.com/tools">
+    <data>
+        <variable
+            name="onboardingFragmentViewModel"
```

Comment:
Why do we need the viewModel here?

Reply:
My bad, I was under the wrong impression that we need to explicitly specify the viewModel inside the fragment layout. I'll remove them